release/models/interfaces/openconfig-if-common.yang

```diff
@@ -0,0 +1,301 @@
+submodule openconfig-if-common {
+
+  yang-version "1";
+
+  belongs-to openconfig-interfaces {
+    prefix "oc-if";
+  }
+
+  import openconfig-yang-types { prefix oc-yang; }
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-types { prefix oc-types; }
+
+  // meta
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    netopenconfig@googlegroups.com";
+
+  description
+    "This sub-module contains groupings that are common across multiple
```

Comment:
This would be a clearer reason to define a new `module` I think, which we could do, since the `uses` statements will still be in the `openconfig-interface` module. But given that this is just one grouping I'm still not sure it is worth it.

Reply:
You're right, the description states a need for what should be a yang module, but in fact this file is a sub-module and is only imported  (as is required by sub-modules) in one 'main' module, openconfig-interfaces.yang.  I've removed the submodule and moved grouping into openconfig-interfaces.yang.

If you have a suggestion on how big a yang file should be before considering breaking it up into multiple files, let me know and I can start a separate PR for the style guide.

FWIW, with the submodule openconfig-interfaces is 965 lines long and contains 20 groupings.  Without the submodule, openconfig-interfaces is 1223 lines long.  It seems ~ 1,000 lines is long enough to consider breaking up into multiple files?  Out of 17 bgp yang models, only 2 have more than 1000 lines.

Reply:
I'm not sure I have a good rule of thumb -- the reason that I created the submodules in BGP when we were doing that (and @3alexb created them in IS-IS) was that there were logical subdivisions that could be made in the module contents that aided in navigation of the overall sets of `groupings`.

In `interfaces` we generally broke things up to add e.g., `ethernet` and `ip` in separate modules -- such that this was the breakdown.